Cargo.lock

```diff
@@ -399,6 +399,26 @@ version = "0.3.3"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "3dcaa9ae7725d12cdb85b3ad99a434db70b468c09ded17e012d86b5c1010f7a7"
 
+[[package]]
+name = "getopts"
```

Comment:
There's not reason why resolving links in the documentation needs command line argument parsing, please enable apropriate `default-feature = false ` to remove unnecessary dependencies.